### ISSUE TITLE
Export: Cannot read property '_knownThemes' of null

### ISSUE THREAD

**Issue by Rob3681** [notable/notable#773]
Since version 1.7.0 I can not export to PDF anymore. The following error message appears in the console:

```
Uncaught (in promise) TypeError: Cannot read property '_knownThemes' of null
    at export_Export.<anonymous> (renderer.js:1)
    at async Object.html (renderer.js:1)
    at async pdf (renderer.js:1)
    at /C:/Users/R/AppData/Local/Programs/notable/resources/app.asar/async file:/C:/Users/R/AppData/Local/Programs/notable/resources/app.asar/renderer.js:1
```

**Comment by fabiospampinato** [comment]
It looks like I missed something, expect a v1.7.1 release soon.

Can you check if adding a code block to the note you're trying to export makes the export succeed?

**Comment by Rob3681** [comment]
Yes. When using a code block, export is possible.

**Comment by fabiospampinato** [comment]
Alright I think I know what the issue is, I'll fix this tomorrow (now it's 2am here). Thanks a lot for reporting the issue!

**Comment by fabiospampinato** [comment]
Actually I don't think the note must contain a code block, it's probably more like you need to render a code block, anywhere in any note, before being able to export anything to PDF currently.

**Comment by fabiospampinato** [comment]
Actually I've just released [v1.7.1](https://github.com/notable/notable/releases/tag/v1.7.1), which addressed this problem. Tomorrow I'll be busy for most of the day and I didn't want to leave you with a broken app, enojy!
